### PR TITLE
Update qodana.yaml to limit target frameworks

### PR DESCRIPTION
**Pull request by hybloid**
Using `frameworks:` property one could limit analysed TFMs.

### CODE REVIEW

**Comment by hybloid** [comment]
I will turn off pr-analysis to verify the report

File: qodana.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ profile:
 
 dotnet:
   solution: FluentAssertions.sln
+  frameworks: net47;net6.0;netstandard2.0;netstandard2.1
```
Comment (IT-VBFK) [inline qodana.yaml:11]: I think we don't want .netstandard2.x to be reported

Comment (dennisdoomen) [reply qodana.yaml]: Correct

Comment (IT-VBFK) [reply qodana.yaml]: I guess we are close to a 0 error baseline report then :D

**Comment by dennisdoomen** [comment]
I'm not sure it works. There are now _no_ problems, which raises my suspicions...

File: qodana.yaml (hunk 1 above)
Comment (hybloid) [reply qodana.yaml]: Folks, need a little bit more time. We've implemented some kind of validation after the limitation feature was implemented. Gonna find the way how to supply the same config in a valid way.

Comment (hybloid) [reply qodana.yaml]: image
image

Comment (hybloid) [reply qodana.yaml]: image

Comment (hybloid) [reply qodana.yaml]: @dennisdoomen the last configuration works (with plus list). We will think how we could supply the minus list then for 241.

**Comment by hybloid** [comment]
@dennisdoomen having

`Using 'local-changes' script as qodana run scenario` 

in the log means that we are running in local changes mode, where we analyze only changed files.

**Comment by dennisdoomen** [comment]
So does this work now?

**Comment by hybloid** [comment]
> So does this work now?

@dennisdoomen it works right now, checked once again and squashed to one commit. You won't see any issues in that PR, since you have pr-mode on. I've double tested it on https://github.com/hybloid/fluentassertions/actions/runs/7460599955

**Comment by dennisdoomen** [comment]
I thought you said that limiting by framework wasn't finished yet.

**Comment by dennisdoomen** [comment]
It works!

Before:

image

After

image

**Comment by jnyrup** [comment]
Thanks for the help @hybloid

**Pull request merged**